spec/models/reporting/current_workload_report_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Reporting::CurrentWorkloadReport do
+  describe '#rows' do
+    subject(:rows) { described_class.new(**arguments).rows }
+
+    let(:arguments) { {} }
+
+    before do
+      travel_to Time.zone.local(2023, 11, 28, 12)
+
+      # rubocop:disable Rails/SkipsModelValidations
+      Review.insert_all([
+                          { application_id: SecureRandom.uuid, business_day: '2023-11-27',
```

Comment:
Small thing just wondering if this would be easier to parse if it was returned from a method

```ruby
def fake_review(bus_day:, work_stream: 'criminal_applications_team', state:)
   { application_id: SecureRandom.uuid, business_day: bus_day,
work_stream: work_stream, state: state}
end
```

base case is CAT team and then you only pass extradition as the WS once.